python/pyarrow/_dataset_parquet.pyx

```diff
@@ -226,7 +226,7 @@ cdef class ParquetFileFormat(FileFormat):
         return f"<ParquetFileFormat read_options={self.read_options}>"
 
     def make_fragment(self, file, filesystem=None,
-                      Expression partition_expression=None, row_groups=None):
+                      Expression partition_expression=None, row_groups=None, size=-1):
```

Comment:
I think you'll want to do something like `max_chunksize` from this code:

https://github.com/apache/arrow/blob/aca1d3eeed3775c2f02e9f5d59d62478267950b1/python/pyarrow/_flight.pyx#L1132

Reply:
Thanks, updated to something like this, so now only the `cdef` functions get -1